tools/build/compile.sh

```diff
@@ -43,7 +43,9 @@ else
 fi
 DEST_PACKAGE_SWIFT="$BASE_PATH/spm-build/Package.swift"
 
-BUILD_FLAGS=""
+# Due to a current bug in the Swift Docker image compile optimization is disabled by default.
+# If you need compiler optimization you can override the BUILD_FLAGS to enable it
+BUILD_FLAGS="-Xswiftc -Onone"
```

Comment:
@csantanapr 
I was thinking of limiting this to solely the Swift 4.1 runtime.  What do you think about that?
I tested that it has no adverse effects on Swift 3.1.1, but I didn't know since it is a Swift 4.1 fix if we should limit it to only that scope.

Reply:
yes add a check to only limit to swift 4 to leave swift3 unchanged.